diff -up ./MgaRepo/rpmutil.py.orig ./MgaRepo/rpmutil.py
--- ./MgaRepo/rpmutil.py.orig	2015-02-20 18:44:13.414834556 +0200
+++ ./MgaRepo/rpmutil.py	2015-02-20 18:43:51.346694254 +0200
@@ -640,18 +640,18 @@ def upload(paths, commit=False):
             topdir = getpkgtopdir()
             binrepo.upload_binary(topdir, os.path.basename(path))
             binrepo.update_sources(topdir, added=[path])
-            if commit:
-                svn = SVN()
-                silent = config.get("log", "ignore-string", "SILENT")
-                message = "%s: new file %s" % (silent, path)
-                svn.commit(binrepo.sources_path(topdir), log=message)
+            #if commit:
+            #    svn = SVN()
+            #    silent = config.get("log", "ignore-string", "SILENT")
+            #    message = "%s: new file %s" % (silent, path)
+            #    svn.commit(binrepo.sources_path(topdir), log=message)
         else:
             svn = SVN()
             svn.add(path, local=True)
-            if commit:
-                silent = config.get("log", "ignore-string", "SILENT")
-                message = "%s: new file %s" % (silent, path)
-                svn.commit(path, log=message)
+            #if commit:
+            #    silent = config.get("log", "ignore-string", "SILENT")
+            #    message = "%s: new file %s" % (silent, path)
+            #    svn.commit(path, log=message)
 
 def delete(paths, commit=False):
     silent = config.get("log", "ignore-string", "SILENT")
@@ -660,14 +660,14 @@ def delete(paths, commit=False):
         if binrepo.is_binary(path):
             topdir = getpkgtopdir()
             binrepo.update_sources(topdir, removed=[os.path.basename(path)])
-            if commit:
-                svn = SVN()
-                svn.commit(binrepo.sources_path(topdir), log=message)
+            #if commit:
+            #    svn = SVN()
+            #    svn.commit(binrepo.sources_path(topdir), log=message)
         else:
             svn = SVN()
             svn.remove(path, local=True)
-            if commit:
-                svn.commit(path, log=message)
+            #if commit:
+            #    svn.commit(path, log=message)
 
 def switch(mirrorurl=None):
     svn  = SVN()
